src/otx/algo/action_classification/backbones/movinet.py

```diff
@@ -10,7 +11,6 @@
 import torch
 import torch.nn.functional as F  # noqa: N812
 from einops import rearrange
```

Comment:
Is `einops` included in otx requirement?

Reply:
added at ab701960e6a342c8fcbbdb6f72e2e61cb24e0ff1